### DISCUSSION TITLE
我登上了绳网！

### DISCUSSION THREAD

**Discussion by fyhwsy** [share121/inter-knot#2659] [其他]
终于可以成为一名绳匠了，这是成为传奇的第一天！
![120287559_p0](https://github.com/user-attachments/assets/a7e7b9cf-7e3d-4495-9781-f7b0589195df)

**Comment by JobNumber-9527** [comment]
月计人特有的无处不在<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
